src/main/java/com/twilio/exception/ApiException.java

```diff
@@ -1,20 +1,23 @@
 package com.twilio.exception;
 
+import java.util.Map;
+
 public class ApiException extends TwilioException {
 
     private static final long serialVersionUID = -3228320166955630014L;
 
     private final Integer code;
     private final String moreInfo;
     private final Integer status;
+    private final Map<String, Object> details;
```

Comment:
Getting this warning from SonarLint: https://rules.sonarsource.com/java/RSPEC-1948

Reply:
I can see the issue, why it is complaining, but the concrete type creation is in Jackson, and i wouldn't want to override that. Fwiw, jackson creates LinkedHashMap<> and its serializable